mmdet/models/roi_heads/htc_roi_head.py

```diff
@@ -173,8 +176,11 @@ def _bbox_forward(self, stage, x, rois, semantic_feat=None):
             bbox_semantic_feat = self.semantic_roi_extractor([semantic_feat],
                                                              rois)
             if bbox_semantic_feat.shape[-2:] != bbox_feats.shape[-2:]:
-                bbox_semantic_feat = F.adaptive_avg_pool2d(
-                    bbox_semantic_feat, bbox_feats.shape[-2:])
+                if torch.onnx.is_in_onnx_export():
+                    bbox_semantic_feat = self.avg_pooling(bbox_semantic_feat)
+                else:
+                    bbox_semantic_feat = F.adaptive_avg_pool2d(
```

Comment:
Guess these are not equivalent.

Reply:
It works only in case if target feature map size is 2 times smaller. With `adaptive_avg_pool2d` the model is not exported to ONNX. So, I removed this patch and suggest to work on it later.